generators/app/templates/e2e/src/_app.e2e-spec.ts

```diff
@@ -6,20 +6,18 @@ import { AppPage } from './app.po';
 describe('app', () => {
   let page: AppPage;
 
-  beforeEach(() => {
+  beforeAll(() => {
```

Comment:
Well these test cases aren't currently independent - it's really one test case, but split in half.  I'd looked and noticed you had a separate issue for overhauling the e2e tests to make them more extensive and be a good example on best practices, etc, so I figured changing the tests too much was outside the scope of this issue.

Because of that, I just did something quick and dirty to eliminate all the extra refreshes, as they were making the ionic version of login fail.  I didn't fully get to the bottom of it, but it seemed to be some kind of race condition.

I don't know how deep you want to go within this issue - another simple approach would be to revert this change, but combine the two it blocks.

Reply:
for this one I'm waiting for a thumbs-up or thumbs-down on the e2e PR I filed.  If you like it and merge it, it'll render these changes obsolete.

if not, let me know and I'll address these here :)

Reply:
I approved your e2e PR, you can update this one after it's merged 😃